Automattic/simplenote-electron
# Move markdown toggle to the toolbar

While working on markdown in the native app, @drw158 and I discussed the current `Edit/Preview` toggle that is displayed in the editor. It takes up quite a bit of vertical space in the editor so we decided to change it to a preview button in the toolbar instead. I'm making that change to this app as well in this PR:

<img width="558" alt="screen shot 2018-01-24 at 12 40 31 pm" src="https://user-images.githubusercontent.com/789137/35355998-06bde62a-0104-11e8-835e-684ed21eda65.png">

<img width="369" alt="screen shot 2018-01-24 at 12 40 16 pm" src="https://user-images.githubusercontent.com/789137/35356007-0a574466-0104-11e8-9131-d96821cb3d10.png">

**To Test**
* Enable markdown for a note, the preview icon should appear.
* Click the icon, it should show the markdown preview and change to a cancel icon
* View the trash, the preview icon should disappear.
* Return to the note, and disable markdown, the icon should disappear.
